tests/run.sh

```diff
@@ -10,11 +10,8 @@ source "${project_directory}/metadata.sh"
 
 # Note: the following environment variables are required by the shared
 # data-services-kudo tooling.
-export DS_KUDO_VERSION="${DS_KUDO_VERSION:-v${KUDO_VERSION}}"
```

Comment:
Why did you remove this one? When this is gone, it basically prevents executing `run.sh` locally, because DS_KUDO_VERSION is undefined.

Reply:
DS_KUDO_VERSION is already set in `metadata.sh`. Setting it here again is redundant.

Reply:
Ah 👍